RadixWallet/Clients/DeviceFactorSourceClient/DeviceFactorSourceClient+Interface.swift

```diff
@@ -46,7 +46,7 @@ extension DeviceFactorSourceClient {
 	public typealias PublicKeysFromOnDeviceHD = @Sendable (PublicKeysFromOnDeviceHDRequest) async throws -> [HierarchicalDeterministicPublicKey]
 	public typealias SignatureFromOnDeviceHD = @Sendable (SignatureFromOnDeviceHDRequest) async throws -> SignatureWithPublicKey
 	public typealias IsAccountRecoveryNeeded = @Sendable () async throws -> Bool
-	public typealias IsSeedPhraseNeededToRecoverAccounts = @Sendable () async throws -> Bool
+	public typealias MissingMnemonicEntities = @Sendable () async throws -> (accounts: [AccountAddress], personas: [IdentityAddress])
```

Comment:
It might be clearer if you reverse the name, to `EntitiesMissingMnemonic`. The current name is ambiguous.

Reply:
we are using `adjectiveNoun` for the rest of the cases:
```
controlledEntities
unrecoverableEntities
```

I would keep it this way or change all.

Reply:
Well, that's because in English you usually put adjectives in front of nouns. But "missing mnemonics" is a present participle used as an attribute, and unlike adjective attributes they almost always go before the noun when the verb has an object. 

You wouldn't say "there are three eating ice cream children in the park", (present participle with object) whereas you would say "there are three small children in the park".

Reply:
But my main point was more that it sounds like this is an array of "mnemonic entities" that are missing...

Reply:
ok, what about `MnemonicMissingEntities`?

Reply:
OK sure. Could still mean "a mnemonic that is missing its entities" but lets use this, then you get your `-Entities` suffix like you wanted, without violating grammar!